Kudu.Core/Infrastructure/OperationClient.cs

```diff
@@ -48,5 +50,13 @@ public OperationClient(ITracer tracer)
                 }
             }
         }
+
+        public static void SkipSslValidation()
```

Comment:
Maybe change the method name to convey that it doesn't always disable it. Like `SkipSslValidationIfNeeded`. Or something less ugly :)